modules/402-ingress-nginx/hooks/safe_daemonset_update.go

```diff
@@ -214,6 +232,20 @@ func daemonSetDeletePodInDs(input *go_hook.HookInput, namespace, dsName string,
 		return nil
 	}
 
+	// it's a reinsurance, to have pods list always in the same order
```

Comment:
The data is always sorted by namespace/name because it is how etcd works. Etcd returns 
 object sorted by the key field, and keys for kubernetes objects are (for example):
 ```
/registry/pods/echoserver/echoserver-6944fb9c86-ph6t7
/registry/pods/echoserver/echoserver-6944fb9c86-w2w6v
/registry/pods/kafka/my-cluster-entity-operator-55b5dc96d9-8pwq9
/registry/pods/kafka/my-cluster-kafka-0
/registry/pods/kafka/my-cluster-zookeeper-0
/registry/pods/kafka/strimzi-cluster-operator-64c46cf88c-7thxg
/registry/pods/kube-system/d8-control-plane-manager-cf4rj
/registry/pods/kube-system/d8-kube-dns-6cff9f994-5x2gd
/registry/pods/kube-system/d8-kube-dns-6cff9f994-jw94w
/registry/pods/kube-system/d8-kube-dns-6cff9f994-tsvtd
/registry/pods/kube-system/etcd-standard-master-0
/registry/pods/kube-system/kube-apiserver-standard-master-0
/registry/pods/kube-system/kube-controller-manager-standard-master-0
 ```
 As long as ETCD preserve this behavior, apiserver will return sorted objects.

Reply:
Remove sort func